### DISCUSSION TITLE
The doc of Amari UE is outdated

### DISCUSSION THREAD

**Discussion by Zplusless** [srsran/srsRAN_Project#405] [General Help] [answered]
Hi, I'm trying to set up srsRAN gNB with Amarisoft UE following the official doc (link: https://docs.srsran.com/projects/project/en/latest/tutorials/source/amariUE/source/index.html)

However, The file "trx_uhd-linux-xxxx-xx-xx.tar.gz" and "trx_driver.h" mentioned in the document is not available in the Amarisoft tar ball downloaded from Amarisoft. It seems that "trx_sdr-linux-2023-03-17.tar.gz" is the correct file name. But there is also a similar file named "trx_ip-linux-2023-03-17.tar.gz". Besides, there is no "trx_driver.h" under "2023-03-17\trx_sdr-linux-2023-03-17". Is it supposed to be "trx_sdr.so"?

all files in "trx_sdr-linux-2023-03-17.tar.gz"
![image](https://github.com/srsran/srsRAN_Project/assets/10062702/ab3eda2a-63ed-4c7a-b4f9-8d4f508d1dbe)

**Comment by Zplusless** [comment, marked as answer]
There are two version of Amarisoft UE. trx_ip_driver and trx_uhd-driver.  trx_uhd_driver is the version used in the document

**Reply by Lab1R531** [reply]
thank you for the answar but it does not work!
I try to install it with the same modalities but the cmake command continue to fail.
I run "cmake ../ [...] -DTRX_DRIVER_DIR=/root/trx_sdr" and the shell output "Could not find TRX_DRIVER_HEADER_PATH using the following files: trx_drive.h".
Can someone help me ?

THank you

**Reply by pgawlowicz** [reply]
@Lab1R531 please follow this tutorial: https://github.com/srsran/srsRAN_Project/tree/main/utils/trx_srsran
